tests/main/interfaces-cups/test-snapd-consumer/meta/snap.yaml

```diff
@@ -0,0 +1,12 @@
+name: test-snapd-consumer
+version: 1
+architectures: [all]
+apps:
+  cups-control:
+    command: bin/sh
+    plugs:
+    - cups-control
+  cups:
+    command: bin/sh
+    plugs:
+    - cups
```

Comment:
I like that there is an app for each plug, it makes the test a good bit simpler, thanks for this!